docs/configuration/unit_testing_rules.md

```diff
@@ -25,6 +25,11 @@ rule_files:
 # optional, default = 1m
 evaluation_interval: <duration>
 
+# optional, default = 0m
```

Comment:
This isn't our usual way of documenting either optional or default options. Can you line it up with the Prometheus configuration docs?